test/sanity/validate-modules/README.rst

```diff
@@ -119,6 +119,8 @@ Errors
 +---------+--------------------------------------------------------------------------------------------------------------------------------------------+
 | 318     | Module deprecated, but DOCUMENTATION.deprecated is missing                                                                                 |
 +---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 319     | ``RETURN`` fragments missing                                                                                                               |
```

Comment:
FYI - This change will end up conflicting with the changes here: https://github.com/ansible/ansible/pull/23925